extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/service/TransferProcessServiceImpl.java

```diff
@@ -72,6 +75,18 @@ public TransferProcessServiceImpl(TransferProcessStore transferProcessStore, Tra
         return apply(transferProcessId, this::deprovisionImpl);
     }
 
+    @Override
+    public @NotNull ServiceResult<String> initiateTransfer(DataRequest request) {
+        return transactionContext.execute(() -> {
+            StatusResult<String> transferInitiateResult = manager.initiateConsumerRequest(request);
+            return Optional.ofNullable(transferInitiateResult)
+                    .filter(AbstractResult::succeeded)
+                    .map(AbstractResult::getContent)
+                    .map(ServiceResult::success)
+                    .orElse(ServiceResult.conflict("Request couldn't be initialised."));
+        });
+    }
+
     private ServiceResult<TransferProcess> apply(String transferProcessId, Function<TransferProcess, ServiceResult<TransferProcess>> function) {
         return transactionContext.execute(() -> {
             var transferProcess = transferProcessStore.find(transferProcessId);
```

Comment:
modifying transfer processes should NOT be done directly through the store, but using the `CommandQueue`, to avoid race conditions. We should already have commands to cancel and deprovision. Could you do this directly in this PR? otherwise lets open another one.

Reply:
There are no modifications done through the `store`, it's used only to validate the "cancellation" and the "deprovisioning", but at the end the actions are executed through the commands

Reply:
ah you're right.